From jason@lakedaemon.net  Wed Nov 12 15:39:55 2008
From: Jason Cooper <jason@lakedaemon.net>
Date: Tue, 11 Nov 2008 13:02:53 -0500
Subject: USB: net: asix: add support for Cables-to-Go USB Ethernet adapter
To: Greg KH <greg@kroah.com>
Cc: <alan@lxorguk.ukuu.org.uk>
Message-ID: <4919C8CD.5020301@lakedaemon.net>


Add support to drivers/net/usb/asix.c for the Cables-to-Go "USB 2.0 to
10/100 Ethernet Adapter". USB id 0b95:772a.

Signed-off-by: Jason Cooper <jason@lakedaemon.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/net/usb/asix.c |    4 ++++
 1 file changed, 4 insertions(+)

--- a/drivers/net/usb/asix.c
+++ b/drivers/net/usb/asix.c
@@ -1444,6 +1444,10 @@ static const struct usb_device_id	produc
 	// Apple USB Ethernet Adapter
 	USB_DEVICE(0x05ac, 0x1402),
 	.driver_info = (unsigned long) &ax88772_info,
+}, {
+	// Cables-to-Go USB Ethernet Adapter
+	USB_DEVICE(0x0b95, 0x772a),
+	.driver_info = (unsigned long) &ax88772_info,
 },
 	{ },		// END
 };
